src/components/RadioButton.js

```diff
@@ -7,12 +7,14 @@ const RadioButton = React.createClass({
     activeButtonStyle: React.PropTypes.object,
     buttonStyle: React.PropTypes.object,
     checked: React.PropTypes.bool,
+    color: React.PropTypes.string,
```

Comment:
Oh you're right it looks like we're using `color` or `primaryColor`. I'll keep an eye on that in the future and switch `primaryColor` to `color` when I see it.